packages/manager/apps/dedicated/client/app/ip/components/list/list.html

```diff
@@ -440,7 +441,14 @@
 
                 <!-- Actions -->
                 <td>
-                    <div data-ng-include="'ip/popover/ipBlock.html'"></div>
+                    <div
+                        data-ng-if="ipBlock.version === IP_TYPE.V6 && (ipBlock.type === FAILOVER || ipBlock.type === VRACK)"
```

Comment:
There are a lot of different ifs at the end so you might consider creating utility function directly on ip.class (on 2API side) to ease the display of the view

Reply:
should do it on dedicated ticket for refactoring or I include it on this PR ?

Reply:
it can be done in a dedicated PR